scripts/static-checks.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+./src/scripts/setup_hsm.sh
+
+mkdir -p build-static-checks
+cd build-static-checks
+cmake -DBUILD_OSTREE=ON -DBUILD_SOTA_TOOLS=ON -DBUILD_DEB=ON -DBUILD_P11=ON -DBUILD_WITH_CODE_COVERAGE=ON ..
```

Comment:
Is building with code coverage here necessary? Seems like it'll just slow the build down.